src/main/scala/com/typesafe/sbt/packager/linux/Keys.scala

```diff
@@ -13,6 +13,7 @@ trait LinuxKeys {
   val daemonGroup = SettingKey[String]("daemon-group", "Group to start application daemon")
   val daemonGroupGid = SettingKey[Option[String]]("daemon-group-gid", "GID of daemonGroup")
   val daemonShell = SettingKey[String]("daemon-shell", "Shell provided for the daemon user")
+  val daemonLogFile = SettingKey[String]("daemon-log-file", "Name of the log file generated by the application daemon")
```

Comment:
Felix and I spoke about this and concluded that it was only needed for rpm. You can't have this choice for Debian as the log is already named with the dpkg-ing and there is no ability to change it.